packages/utilities/fast-web-utilities/src/numbers.ts

```diff
@@ -20,3 +20,15 @@ export function wrapInBounds(min: number, max: number, value: number): number {
 export function limit(min: number, max: number, value: number): number {
     return Math.min(Math.max(value, min), max);
 }
+
+/**
+ * Determines if a number value is within a specified range.
+ *
+ * @param value - the value to check
+ * @param min - the range start
+ * @param max - the range end
+ */
+export function inRange(value: number, min: number, max: number = 0): boolean {
+    [min, max] = [min, max].sort((a, b) => a - b);
+    return min <= value && value < max;
```

Comment:
Sorting is expensive. Have you considered using the limit function to verify that the value is in range?

```suggestion
    return limit(Math.min(min, max), Math.max(min, max), value) === value;
```

Reply:
This fails the test "should return `false` when `value` is equal to `max`".

Reply:
So the logic is that it can be equal or greater than min or less than max? Shouldn't min and max logic be handled the same? It seems a little confusing this way.

Reply:
My thinking was to match Lodash's `inRange` behavior, which handles `min` and `max` this way:

> Checks if `n` is between `start` and up to, but not including, `end`. If `end` is not specified, it's set to `start` with `start` then set to `0`. If `start` is greater than `end` the params are swapped to support negative ranges.